eos/form-factors/parametric-kkvdz2022.cc

```diff
@@ -48,8 +51,24 @@ namespace eos
                  UsedParameter(p[_par_name("rho_3_2")],  *this) },
                { UsedParameter(p[_par_name("rho_4_0")],  *this),
                  UsedParameter(p[_par_name("rho_4_1")],  *this),
-                 UsedParameter(p[_par_name("rho_4_2")],  *this) }}}
-    {
+                 UsedParameter(p[_par_name("rho_4_2")],  *this) }}},
+        _c_subtraction{{{ UsedParameter(p[0.0],  *this),
```

Comment:
I had no idea this worked. Why are you looking up a parameter by index. Even more so, why are you using a ``float`` as the index?